Sources/krink/eventhandler.c

```diff
@@ -166,26 +167,33 @@ static void shutdown(void) {
 	notify(KR_EVT_SHUTDOWN, unused);
 }
 
+static void drop_files(wchar_t* file,void* udata){
```

Comment:
please make sure you run clang-format or manually adhere to the formatting specified